apps/backend/src/utils/Cache.ts

```diff
@@ -23,8 +23,15 @@ export class Cache<T> implements ICache<T> {
   private removals = 0;
 
   public constructor(maxEntries: number, secondsToLive?: number) {
-    this.maxEntries = maxEntries;
-    this.millisecondsToLive = secondsToLive ? secondsToLive * 1000 : 10000;
+    this.maxEntries =
+      Number(process.env.CACHE_MAX_ENTRIES) > maxEntries
+        ? Number(process.env.CACHE_MAX_ENTRIES)
+        : maxEntries;
+    this.millisecondsToLive = secondsToLive
+      ? Number(process.env.CACHE_TTL) > secondsToLive
+        ? Number(process.env.CACHE_TTL) * 1000
+        : secondsToLive * 1000
+      : 10000;
```

Comment:
It feels odd to me to have override parameters that will only be used if they're larger. And this layout also means that the CACHE_TTL variable will be ignored if the constructor is called without a secondsToLive parameter. I'd have thought the logic might go something like `this.millisecondsToLive = (Number(process.env.CACHE_TTL ?? secondsToLive ?? 10)) * 1000;`

And could I be inconvenient and ask that CACHE_TTL be renamed to include SECONDS so it's immediately clear in the config what units it uses